backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java

```diff
@@ -643,54 +646,21 @@ public static String getJoinedLocation(List<String> location) {
     return String.join("#", location);
   }
 
-  public interface RetryCallInterface<T> {
-    T retryCall(boolean retry);
-  }
-
-  public static Object retryOrThrowException(
-      StatusRuntimeException ex, boolean retry, RetryCallInterface<?> retryCallInterface) {
-    String errorMessage = ex.getMessage();
-    LOGGER.debug(errorMessage);
-    if (ex.getStatus().getCode().value() == Code.UNAVAILABLE_VALUE) {
-      errorMessage = "UAC Service unavailable : " + errorMessage;
-      if (retry && retryCallInterface != null) {
-        try {
-          App app = App.getInstance();
-          Thread.sleep(app.getRequestTimeout() * 1000);
-          retry = false;
-        } catch (InterruptedException e) {
-          Status status =
-              Status.newBuilder()
-                  .setCode(Code.INTERNAL_VALUE)
-                  .setMessage("Thread interrupted while UAC retrying call")
-                  .build();
-          throw StatusProto.toStatusRuntimeException(status);
-        }
-        return retryCallInterface.retryCall(retry);
-      }
-
-      Status status =
-          Status.newBuilder().setCode(Code.UNAVAILABLE_VALUE).setMessage(errorMessage).build();
-      throw StatusProto.toStatusRuntimeException(status);
-    }
-    throw ex;
-  }
-
   public static void initializeBackgroundUtilsCount() {
     int backgroundUtilsCount = 0;
     try {
-      if (System.getProperty(ModelDBConstants.BACKGROUND_UTILS_COUNT) == null) {
```

Comment:
this `initializeBackgroundUtilsCount` function should be in commin

Reply:
moved